packages/main/src/services/local-library/index.ts

```diff
@@ -62,7 +62,12 @@ class LocalLibrary {
       path: file,
       folder,
       duration: format.duration,
-      name: common.title,
+      name: function() {
+        if (common.title !== undefined && common.title != null) {
```

Comment:
can you replace this condition with `_.isNil(common.title)`?

Reply:
Of course I can but that would check only for `null`, not when the variable is just `undefined`,
when I replace the whole condition, it does this:
```
@nuclear/main:       main › QueryFailedError: SQLITE_CONSTRAINT: NOT NULL constraint failed: local_track.name
@nuclear/main:     at new QueryFailedError (webpack:///./node_modules/typeorm/error/QueryFailedError.js?:11:28)
@nuclear/main:     at Statement.handler (webpack:///./node_modules/typeorm/driver/sqlite/SqliteQueryRunner.js?:48:38)
```
Maybe you wanted `common.title !== undefined &&  _.isNil(common.title) ` ?